component/src/main/java/io/siddhi/extension/util/Constant.java

```diff
@@ -30,4 +30,8 @@
     public static final String SFTP_SCHEME_FILE_OPTION = "sftp";
     public static final String VFS_SCHEME_KEY = "VFS_SCHEME";
     public static final String FTP_SCHEME_FILE_OPTION = "ftp";
+    public static final String STATUS_DONE = "Completed";
+    public static final String STATUS_PROCESS = "Copying";
+    public static final String STATUS_NEW = "Newly Added";
```

Comment:
Shall we make as "created"

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/pull/82/commits/3f9a2a0eb50655821dd322688aa034be33ef8860